src/nvim/tui/terminfo_defs.h

```diff
@@ -1,6 +1,6 @@
 // uncrustify:off
 
-// Generated by scripts/update_terminfo.sh and ncurses 6.4.20230520
+// Generated by scripts/update_terminfo.sh and ncurses 6.3.20211021
```

Comment:
Why is it still 6.3 here if you generated with updated ncurses? Although I ran `./scripts/update_terminfo.sh` on your branch and the only change is the version here.

Reply:
Honestly i don't know. I am using wsl to run the script, maybe something is wrong there. i will regenerate again with cygwin

Reply:
Hmm that seems to cause more problems. Let's revert the newest commit then.